themes/bootstrap3/js/combined-search.js

```diff
@@ -0,0 +1,22 @@
+/*global VuFind, setupOpenUrlLinks, checkSaveStatuses, setupSaveRecordLinks */
+VuFind.combinedSearch = (function() {
```

Comment:
I really like this model as well. After this branch is merged, I'll switch the VuFind namespace to this model for consistency. Sound good?